pkg/scheduler/eventhandlers.go

```diff
@@ -407,7 +441,8 @@ func addAllEventHandlers(
 
 	informerFactory.Storage().V1().StorageClasses().Informer().AddEventHandler(
 		cache.ResourceEventHandlerFuncs{
-			AddFunc: sched.onStorageClassAdd,
+			AddFunc:    sched.onStorageClassAdd,
+			UpdateFunc: sched.onStorageClassUpdate,
```

Comment:
So updating a StorageClass can also make some pods schedulable?

Reply:
yes, in volume binding we [check if the node can satisfy the topology requirements of the class](https://github.com/kubernetes/kubernetes/blob/a8035dbc3b1a523932f8e2384b5f49ace4194870/pkg/controller/volume/scheduling/scheduler_binder.go#L901-L905), and the allowed topologies are mutable.